lib/src/common/model/checks.dart

```diff
@@ -362,12 +362,14 @@ class CheckRunAction {
 @immutable
 class CheckSuite {
   final int? id;
+  final String? headBranch;
```

Comment:
Can you update the changelog and pubspec?

Reply:
Done! I wasn't sure about the version to use so lmk